glean-core/android/metrics.yaml

```diff
@@ -254,4 +254,4 @@ glean.core.migration:
     notification_emails:
       - glean-team@mozilla.com
       - aplacitelli@mozilla.com
-    expires: 2020-03-01
+    expires: 2020-03-31
```

Comment:
This collection still needs data review, yes?

Reply:
Oh, hey, I see it in my queue.

Here's the url: https://bugzilla.mozilla.org/show_bug.cgi?id=1582099#c4